crypto/src/signature.rs

```diff
@@ -132,3 +132,56 @@ pub mod json {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::signature::json::signature_type::SignatureTypeJson;
+    use crate::signature::json::{SignatureJson, SignatureJsonRef};
```

Comment:
I can be slightly more readable to use `super`:

```suggestion
    use super::json::signature_type::SignatureTypeJson;
    use super::json::{SignatureJson, SignatureJsonRef};
```